junifer/api/functions.py

```diff
@@ -270,8 +269,7 @@ def queue(
 
     yaml_config = jobdir / "config.yaml"
     logger.info(f"Writing YAML config to {str(yaml_config.absolute())}")
-    with open(yaml_config, "w") as f:
-        f.write(yaml.dump(config))
+    yaml.dump(config, stream=yaml_config)
```

Comment:
Why does this not need a context manager anymore?

Reply:
That's cause `ruamel.yaml` allows you to write to files via `stream` argument in the `dump` function.

Reply:
and this takes care of closing the ressource?